src/xcode/ENA/ENA/Source/Services/HealthCertificate/Models/Extensions/HealthCertificate+PDF/HealthCertificate+PDF.swift

```diff
@@ -0,0 +1,338 @@
+				color: textColor,
+				font: HealthCertificate.openSansFont,
+				rect: CGRect(x: 29, y: 694, width: 266, height: 23),
+				upsideDown: false
+			),
+			PDFText(
+				text: dateOfBirth,
+				color: textColor,
+				font: HealthCertificate.openSansFont,
+				rect: CGRect(x: 29, y: 740, width: 266, height: 23),
+				upsideDown: false
+			),
+			PDFText(
+				text: entry.uniqueCertificateIdentifier.removeURNPrefix(),
+				color: textColor,
+				font: HealthCertificate.openSansFont,
+				rect: CGRect(x: 29, y: 785, width: 266, height: 23),
+				upsideDown: false
+			)
+			
+		]
+	}
+
+	private func recoveryTexts(for entry: RecoveryEntry, with valueSets: SAP_Internal_Dgc_ValueSets) -> [PDFText] {
+		[
+			PDFText(
+				text: entry.certificateValidUntil,
+				color: textColor,
+				font: HealthCertificate.openSansFont,
+				rect: CGRect(x: 14, y: 56, width: 125, height: 47),
+				upsideDown: true
+			),
+			PDFText(
+				text: entry.certificateValidFrom,
+				color: textColor,
+				font: HealthCertificate.openSansFont,
+				rect: CGRect(x: 14, y: 118, width: 125, height: 23),
+				upsideDown: true
+			),
+			PDFText(
+				text: entry.certificateIssuer,
+				color: textColor,
+				font: HealthCertificate.openSansFont,
+				rect: CGRect(x: 14, y: 163, width: 125, height: 23),
+				upsideDown: true
+			),
+			PDFText(
+				text: entry.countryOfTest,
+				color: textColor,
+				font: HealthCertificate.openSansFont,
+				rect: CGRect(x: 14, y: 209, width: 125, height: 23),
+				upsideDown: true
+			),
+			PDFText(
+				text: entry.dateOfFirstPositiveNAAResult,
+				color: textColor,
+				font: HealthCertificate.openSansFont,
+				rect: CGRect(x: 14, y: 263, width: 125, height: 19),
+				upsideDown: true
+			),
+			PDFText(
+				text: entry.formattedValue(for: \RecoveryEntry.diseaseOrAgentTargeted, valueSets: valueSets) ?? "",
+				color: textColor,
+				font: HealthCertificate.openSansFont,
+				rect: CGRect(x: 14, y: 326, width: 125, height: 28),
+				upsideDown: true
+			),
+			PDFText(
+				text: name.reversedFullNameWithoutFallback,
+				color: textColor,
+				font: HealthCertificate.openSansFont,
+				rect: CGRect(x: 28, y: 694, width: 266, height: 23),
+				upsideDown: false
+			),
+			PDFText(
+				text: dateOfBirth,
+				color: textColor,
+				font: HealthCertificate.openSansFont,
+				rect: CGRect(x: 28, y: 740, width: 266, height: 23),
+				upsideDown: false
+			),
+			PDFText(
+				text: entry.uniqueCertificateIdentifier.removeURNPrefix(),
+				color: textColor,
+				font: HealthCertificate.openSansFont,
+				rect: CGRect(x: 28, y: 785, width: 266, height: 23),
+				upsideDown: false
+			)
+		]
+	}
+
+	private static var openSansFont: UIFont = {
+		guard let font = UIFont(name: "OpenSans-Regular", size: 8) else {
+			fatalError("Could not find OpenSans-Font.")
+		}
+		return font
+	}()
+
+	private var textColor: UIColor {
+		.enaColor(for: .certificatePDFBlue)
+	}
+}
+
+private extension String {
+
+	func removeURNPrefix() -> String {
```

Comment:
Just a tiny naming suggestion, as it is a nonmutating function

```suggestion
	func removingURNPrefix() -> String {
```